### DISCUSSION TITLE
Microsoft.WindowsDesktop.App 6.0.12

### DISCUSSION THREAD

**Discussion by taipignas** [gitextensions/gitextensions#10659] [General] [answered]
Not sure where to put this.
Maybe it would be better not to rush with the latest versions so fast?
I have latest VS2022 Preview installed and its installing Microsoft.WindowsDesktop.App 6.0.12 only, while the portable version asks for Microsoft.WindowsDesktop.App 6.0.13.
I think nothing bad would happen if nuget would be updated not in 5 days(6.0.13 release date is Jan10, 4.0.2 release date Jan15) but lets say 2 months.

**Comment by vbjay** [comment]
We pull latest release of 6.0 in our cli. Visual studio builds I'm sure have to freeze their version at some point to gather dependencies. 6.0.* should not be a breaking change. Frankly. I use chocolatey to keep my sdk / runtime version to latest.  Installing a later same major/minor version should not be a problem. We set it to 6.0 in code base so it can be developed with your current sdk version. The build pipeline alters the correct files to use latest release of 6.0 when code is pushed up.  

See 


https://github.com/gitextensions/gitextensions/blob/master/appveyor.yml#L21

https://github.com/gitextensions/gitextensions/blob/master/appveyor.yml#L41-L44

**Comment by RussKie** [comment, marked as answer]
The app targeting the latest *runtime* (at the time of build), as generally
that contains the latest security fixes (
https://dotnet.microsoft.com/en-us/download/dotnet/6.0).

For development purposes you can use any version of .NET 6 *SDK* (6.0.2+).

The SDK and the runtime are two different things, and shouldn't be
confused.

**Comment by taipignas** [comment]
Im working on a restricted machine and i cannot update runtimes whenever i want. I will just stay couple of version back then. Thanks for the clarification.

**Reply by RussKie** [reply]
As a workaround you can dial down the version in *.runtimeconfig.json.
